pkg/plugin/manager/manager.go

```diff
@@ -75,12 +79,20 @@ func (m *manager) RunMigrations() error {
 	if len(paths) < 1 {
 		return nil
 	}
+
 	// Init directory for temporary copies of migrations
 	err = m.setupMigrationEnv()
 	if err != nil {
 		return err
 	}
 	defer m.cleanUp()
+
+	// First run the public schema migrations
+	err = m.runPublicMigrations()
+	if err != nil {
+		return err
+	}
+
 	// Creates copies of migrations for all the plugin's transformers in a tmp dir
 	err = m.createMigrationCopies(paths)
```

Comment:
Not your responsibility but I feel like this function should be renamed since it actually, you know, _runs_ migrations

Reply:
Same. I tried to pull the running of migrations out of it, but it didn't work, a rename is a simple solution.